usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json

```diff
@@ -45,18 +45,18 @@
     "  COALESCE(transaction_fpds.place_of_perform_country_n, transaction_fabs.place_of_perform_country_n) AS pop_country_name,",
     "  COALESCE(transaction_fpds.place_of_perform_country_c, transaction_fabs.place_of_perform_country_c, 'USA') AS pop_country_code,",
     "  COALESCE(transaction_fpds.place_of_performance_state, transaction_fabs.place_of_perfor_state_code) AS pop_state_code,",
-    "  COALESCE(transaction_fpds.place_of_perform_county_co, transaction_fabs.place_of_perform_county_co) AS pop_county_code,",
+    "  LPAD(CAST(CAST((REGEXP_MATCH(COALESCE(transaction_fpds.place_of_perform_county_co, transaction_fabs.place_of_perform_county_co), '^[A-Z]*(\\d+)(?:\\.\\d+)?$'))[1] AS smallint) AS text), 3, '0') AS pop_county_code,",
```

Comment:
Unfortunately for changes like this, they need to be propagated to all other materialized views which contain the same data field.

I believe all of these contain the same fields:
SubawardView, SummaryTransactionGeoView, SummaryTransactionMonthView, SummaryTransactionView,
ContractAwardSearchMatview, DirectPaymentAwardSearchMatview, GrantAwardSearchMatview, IDVAwardSearchMatview, LoanAwardSearchMatview, OtherAwardSearchMatview, Pre2008AwardSearchMatview